docs/pages/blog/mui-x-v6-alpha-zero.md

```diff
@@ -0,0 +1,96 @@
+You can rest assured that all MUI X v6 components will be fully compatible with MUI Core v5.
+::::
+
+## How is it going to work?
+
+Before officially releasing v6, we'll go through a few months with pre-releases. We’ll follow the current weekly release process and [semver versioning](https://semver.org/); The difference now is that instead of v5, we’ll ship v6 pre-release packages.
+
+First, in the alpha phase, we’ll introduce all the breaking changes planned for this major. We’ll be exploring a bit, so it’s expected that some APIs will be unstable, not in terms of functionality, but we may need to rename or adjust parameters. This phase is planned to take about two months.
+
+Next, comes the beta phase, where the APIs are more stable, and we focus on bug fixing and polishing the hard edges. This phase is planned to take about one month.
+
+:::info
+Disclaimer: We're working with 2 and 1 months for the alpha and beta phase respectively, but that's just a **reference**.
+
+We operate in a dynamic environment that's subject to change. We may need to do more iterations or change the referenced time windows to deliver the great product we want.
+:::
+
+At last, we’ll release the first `v6.0.0` stable, and we’ll continue improving the components and adding features, but without making any new breaking changes until the next major version.
+We’re establishing a one-year cycle for major versions, meaning the next major comes about one year after this release.
+
+## What happens to v5?
+
+During v6 pre-releases, v5 is the official current major, and it will remain supported during the next version development.
+However, we’ll release new v5 versions by demand, only to patch bugs, or add community contributions.
+
+All new features and enhacements go to v6, and after its first stable release, v5 is officially in [Long-term support](https://mui.com/versions/#long-term-support-lts).
+
+## Where's the v6 documentation?
+
+As mentioned above, v5 is still the official version, so by default, the documentation shows v5 features and API.
+The next version's documentation is in the `next` subdomain.
+
+- [https://next.mui.com/x/react-data-grid/](https://next.mui.com/x/react-data-grid/)
+- [https://next.mui.com/x/react-date-pickers/](https://next.mui.com/x/react-date-pickers/)
+
+## What’s coming next?
+
+The following is a list of enhancements on the pipeline for the next few months. It contains the highlights that will make into the first versions.
+
+### Data Grid
+
+- **Extended customization abilities**.  
+   We want to empower more users to suit the data grid to complex use cases.
+
+  - [ApiRef in the community package](https://github.com/mui/mui-x/issues/6147).
+  - [Filtering on header](https://github.com/mui/mui-x/issues/6247).
+  - [Use the Data Grid internal components outside the grid](https://github.com/mui/mui-x/issues/2522).
+
+- **Improved Design**  
```

Comment:
Design could be confused with design in the general sens, like design architecture

```suggestion
- **Improved look & feel**  
```

Reply:
But maybe it's ok
```suggestion
- **Improved design**  
```